src/vos/tests/evt_ctl.c

```diff
@@ -1078,36 +1201,42 @@ ts_cmd_run(char opc, char *args)
 {
 	int	rc;
 
+	tst_fn_val.optval = args;
+	tst_fn_val.input = true;
+
 	switch (opc) {
 	case 'C':
-		rc = ts_open_create(true, args);
+		rc = run_create_tests();
```

Comment:
Minor: It's not really more than one test we are running so run_create_test would be more appropriate